@kiva/kv-analytics/src/index.js

```diff
@@ -11,6 +12,22 @@ export default {
 		let fbLoaded;
 		const queue = new SimpleQueue();
 
+		Vue.mixin({
```

Comment:
I'm not sure we're going to need this mixin as the global activation will likely need to be different here. For Ui and potentially other apps the headScripts actually need to come down in the html from the server.

Reply:
If we remove this Vue.mixin block I think the rest of this PR can be merged and considered done. This mixin approach on mounted isn't the correct location to initialize the head scripts across different apps. We can create additional tickets for "using" this new package in each app that needs it.